lib/tasks/complexity.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var complexity = require('../pipelines/complexity');
+var normalizeOptions = require('../options');
+var path = require('path');
```

Comment:
Running `npm run lint` throws an error due to this not being used. Can you remove it?